source/tunneling/SecureTunnelingFeature.cpp

```diff
@@ -34,15 +33,13 @@ namespace Aws
                 SecureTunnelingFeature::~SecureTunnelingFeature() { aws_http_library_clean_up(); }
 
                 int SecureTunnelingFeature::init(
-                    shared_ptr<SharedCrtResourceManager> sharedCrtResourceManager,
-                    shared_ptr<ClientBaseNotifier> notifier,
+                    const shared_ptr<SharedCrtResourceManager> sharedCrtResourceManager,
```

Comment:
It has been too long since I made the change so I do not remember... I was experimenting with some linter that probably told me to do this. I removed it and changed the code back and it works so will revert.